app/views/sessions/new.html.erb

```diff
@@ -11,6 +11,11 @@
       <%= f.label :password %>
       <%= f.password_field :password, class: 'form-control' %>
 
+      <%= f.label :remember_me, class: "checkbox inline" do %>
```

Comment:
Would this be safe in the event that someone logged in on a public computer? Would you just assume that they would know to logout if they didn't want their session to be remembered or include some sort of message?